packages/api/Tests/UnitTests/Flexihours/RegisterPaidOvertimeTests.cs

```diff
@@ -66,6 +67,95 @@ public void GetRegisteredPayouts_Registered3Times_ListWith5Items()
             Assert.Equal(3, registeredPayouts.Entries.Count());
         }
 
+        [Fact]
+        public void RegisterPayout_CalculationCorrectForBeforeAndAfterCompRate()
+        {
+            _context.Hours.Add(CreateTimeEntry(date: new DateTime(2020, 01, 02), value: 10M, out int taskid));
+            _context.CompensationRate.Add(CreateCompensationRate(taskid, 2.0M));
```

Comment:
Se på mulighet for å lage en større "akseptanse test" som inneholder en "normal brukerflyt" via API-et, ala:
1. Føre overtid
2. Flekse deler av overtiden
3. Hente ut fleks og se at denne er korrekt
4. Utbetale deler av overtiden
5. Hente ut utbetaling og se at denne er korrekt
6. Hente ut tilgjengelig tid og se at denne er korrekt.


Trenger ikke gjøre dette som del av denne PR, men konge om du gidder å registrere et issue!